ako-infra/avirest/handle_netinfo.go

```diff
@@ -67,8 +67,14 @@ func SyncLSLRNetwork() {
 	if !matched {
 		cloudModel.NsxtConfiguration.DataNetworkConfig.Tier1SegmentConfig.Manual.Tier1Lrs = constructLsLrInCloud(lslrmap)
 		path := "/api/cloud/" + *cloudModel.UUID
-		restOp := utils.RestOp{ObjName: utils.CloudName, Path: path, Method: utils.RestPut, Obj: &cloudModel,
-			Tenant: "admin", Model: "cloud", Version: utils.CtrlVersion}
+		restOp := utils.RestOp{
+			ObjName: utils.CloudName,
+			Path:    path,
+			Method:  utils.RestPut,
+			Obj:     &cloudModel,
+			Tenant:  "admin",
```

Comment:
@chauhanshubham: One question: Not related to this PR: For AKO-infra objects, are we always use `admin` tenant?